etc/conf.templates/standalone/20-warpscript.conf.template

```diff
@@ -85,8 +85,8 @@ warpscript.maxdepth.hard = 2000
 // Maximum number of datapoint which can be fetched during a WarpScript execution
 // Can be modified by LIMIT up to the hard limit below
 //
-warpscript.maxfetch = 100000
-warpscript.maxfetch.hard = 1000000
+warpscript.maxfetch = 1000000
+warpscript.maxfetch.hard = 10000000
```

Comment:
That might be a little too high as a default value, maybe consider lowering it to 2000000 instead